src/components/Editor/Status.vue

```diff
@@ -89,22 +89,19 @@ export default {
 			return this.dirtyStateIndicator ? t('text', 'Saving …') : t('text', 'Saved')
 		},
 		dirtyStateIndicator() {
-			return this.dirty || this.hasUnsavedChanges
+			return this.dirty || this.document.hasUnsavedChanges
```

Comment:
Does `document` really have that function? I think you defined it for the `connection` below.

Reply:
Good catch. Changed this to `this.$syncService.hasUnsavedChanges` in all places now.